README.md

```diff
@@ -8,14 +8,36 @@ This directory contains the necessary files to build a Red Hat compatible contai
 - `llama` CLI tool installed: `pip install llama-stack`
 - Podman or Docker installed
 
+## Build Modes
+
+The build script supports three modes:
+
+### 1. Full Mode (Default)
+Includes all features including TrustyAI providers that require Kubernetes/OpenShift:
+```bash
+./distribution/build.py
+```
+
+### 2. Standalone Mode
+Builds a version without Kubernetes dependencies, using Llama Guard for safety:
+```bash
+./distribution/build.py --standalone
+```
+
+### 3. Unified Mode (Recommended)
+Builds a single container that supports both modes via environment variables:
+```bash
+./distribution/build.py --unified
+```
```

Comment:
Why would we add this level of complexity rather than just always doing a Unified mode?